### ISSUE TITLE
Plugin fails to build (4.x, windows)

### ISSUE THREAD

**Issue by MatteoPiovanelli** [samdze/godot-native-bullets-plugin#21]
I'm building the plugin on my Windows machine.

My compilation Pipeline uses VIsual Studio 2022 (rather, its build tools) and scons 4.7.

I have several errors where private members of `Bullet` are being referenced.

Many of those instances can be "fixed" by making those members as `protected` rather than `private`. In some cases I'm having to invoke the corresponding getters/setter. (I've actually added a method to increase the value of `lifetime` directly.

Does that make sense? I'll draft a PR once I'm done and I have something that builds and looks to perhaps work right.

**Comment by samdze** [comment]
Yes it is an issue that has been reported before but was fixed by the same user by changing its toolchain on Windows.
Sure, you can draft a PR against the 4.x branch with the access specifiers changes if you want
